src/components/FindMovie/FindMovie.tsx

```diff
@@ -20,38 +108,48 @@ export const FindMovie: React.FC = () => {
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {(isError && !isTyping) && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={cn(
```

Comment:
Don't cut variable names
```suggestion
              className={classNames(
```